rootfs/usr/local/bin/helmfile-directory

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
```

Comment:
I think this should be generalized to config-directory and take an argument which is the project (e.g Helmfile)

Reply:
Replaced with `conf-directory helmfiles`